src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -919,23 +913,23 @@ namespace winrt::TerminalApp::implementation
         auto const shiftDown = WI_IsFlagSet(CoreWindow::GetForCurrentThread().GetKeyState(winrt::Windows::System::VirtualKey::Shift), CoreVirtualKeyStates::Down);
 
         winrt::Microsoft::Terminal::Control::KeyChord kc{ ctrlDown, altDown, shiftDown, static_cast<int32_t>(key) };
-        const auto actionAndArgs = _settings.KeyMap().TryLookup(kc);
-        if (actionAndArgs && (actionAndArgs.Action() == ShortcutAction::CloseTab || actionAndArgs.Action() == ShortcutAction::NextTab || actionAndArgs.Action() == ShortcutAction::PrevTab || actionAndArgs.Action() == ShortcutAction::ClosePane))
+        const auto cmd{ _settings.ActionMap().GetActionByKeyChord(kc) };
+        if (cmd && (cmd.Action().Action() == ShortcutAction::CloseTab || cmd.Action().Action() == ShortcutAction::NextTab || cmd.Action().Action() == ShortcutAction::PrevTab || cmd.Action().Action() == ShortcutAction::ClosePane))
```

Comment:
we gotta do something about `Action().Action()`, because that's mental to me. Plus, given how frequently we interchange `cmd`, `args`, `action`, it's probably best to make sure each of the layers has a unique property name, so we immediately know what `foo.Action()` is

Reply:
Changed to `command.ActionAndArgs().Action()`